File: src/main/java/com/developersam/web/control/chunkreader/TestServlet.java
package com.developersam.web.control.chunkreader;

import com.google.cloud.language.v1beta2.Document;
import com.google.cloud.language.v1beta2.Document.Type;
import com.google.cloud.language.v1beta2.LanguageServiceClient;
import com.google.cloud.language.v1beta2.Sentiment;

import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

/**
 * Run a sentiment analysis on "Hello World".
 */
@WebServlet(name = "TestServlet", value = "/apis/chunkreader/test")
public class TestServlet extends HttpServlet {
    
    @Override
    protected void doGet(HttpServletRequest req, HttpServletResponse resp) {
        try (LanguageServiceClient language = LanguageServiceClient.create()) {
            // The text to analyze
            String text = "Hello, world!";
            Document doc = Document.newBuilder()
                    .setContent(text).setType(Type.PLAIN_TEXT).build();
            // Detects the sentiment of the text
            Sentiment sentiment = language.analyzeSentiment(doc)
                    .getDocumentSentiment();
            resp.getWriter().print(sentiment.getScore() + " "
                    + sentiment.getMagnitude());
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
    
}

File: src/main/kotlin/com/developersam/web/controller/chunkreader/TestServlet.kt
package com.developersam.web.controller.chunkreader

import com.developersam.web.model.chunkreader.NLPAPIAnalyzer

import javax.servlet.annotation.WebServlet
import javax.servlet.http.HttpServlet
import javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletResponse

/**
 * Run a sentiment analysis on "Hello World".
 */
@WebServlet("/apis/chunkreader/test")
class TestServlet : HttpServlet() {

    override fun doGet(req: HttpServletRequest, resp: HttpServletResponse) {
        val analyzer = NLPAPIAnalyzer.analyze("Four score and seven " +
                "years ago, I created the matrix")
        resp.writer.println(analyzer?.sentiment)
        resp.writer.println(analyzer?.entities)
        resp.writer.println(analyzer?.sentences)
        resp.writer.println(analyzer?.categories)
    }

}